securityhub-forwarder/sam/packaged.yaml

```diff
@@ -4,26 +4,6 @@ Description: 'This function is invoked by Sumo Logic(via Scheduled Search) throu
   API Gateway. The event payload received is then forwarded to AWS Security Hub.
 
   '
-Metadata:
```

Comment:
It seems this secton (AWS::ServerlessRepo::Application) has been removed unintentionally